app/controllers/catalog_controller.rb

```diff
@@ -77,13 +77,13 @@ class CatalogController < ApplicationController
     # :show may be set to false if you don't want the facet to be drawn in the
     # facet bar
     config.add_facet_field 'genre_pul_label_facet', label: 'Genre'
-    config.add_facet_field 'category_facet', label: 'Category', show: false
-    config.add_facet_field 'subject_label_facet', label: 'Subject', show: false
-    config.add_facet_field 'language_label_facet', label: 'Language'
-    config.add_facet_field 'geographic_subject_label_facet', label: 'Geographic Subject'
-    config.add_facet_field 'geographic_origin_label_facet', label: 'Geographic Origin'
     config.add_facet_field 'date_created_facet', label: 'Date Created', range: true
+    config.add_facet_field 'geographic_origin_label_facet', label: 'Geographic Origin'
     config.add_facet_field 'category_subject_facet', label: 'Subjects', pivot: ['category_facet', 'subject_label_facet']
+    config.add_facet_field 'geographic_subject_label_facet', label: 'Geographic Subject'
+    config.add_facet_field 'language_label_facet', label: 'Language'
+    config.add_facet_field 'category_facet', label: 'Category', show: false
+    config.add_facet_field 'subject_label_facet', label: 'Subject', show: false
```

Comment:
Do we want these separate from the Category/Subject pivot facet above?

Reply:
Yeah I am not sure; they're `show:false` so I'm not sure how they're being used exactly.

Reply:
I think it's an implementation trick to get the pivot facet to work?

Reply:
👍  that makes sense.